src/services/fetchers/RandomNumberFetcher.ts

```diff
@@ -3,10 +3,10 @@ import {inject, injectable} from 'inversify';
 import {BaseProvider} from '@ethersproject/providers';
 import {ChainsIds} from '../../types/ChainsIds.js';
 import {ProviderRepository} from '../../repositories/ProviderRepository.js';
-import {FeedFetcherInterface, FeedFetcherOptions} from 'src/types/fetchers.js';
+import {FeedFetcherOptions} from 'src/types/fetchers.js';
 
 @injectable()
-class RandomNumberFetcher implements FeedFetcherInterface {
+class RandomNumberFetcher {
```

Comment:
why this is reverted if it was already under correct interface?

Reply:
The interface was returning an `any`, now returns a concrete type `FetcherResut`. If this fetcher is `FeedFetcherInterface` then we need to return a new composed type only because of this particular fetcher. What do you think? do it make sense to modify `FetcherResult` only because this fetcher?

Reply:
ok to speed up lets leave it